.github/workflows/release.yml

```diff
@@ -24,68 +24,27 @@ jobs:
       - name: Workaround for https://github.com/actions/checkout/pull/697
         run: git fetch --force origin $(git describe --tags):refs/tags/$(git describe --tags)
 
-      - name: Get version number
-        run: git describe | tee version
-
-      - name: Extract tag body
-        run: git tag --points-at "${GITHUB_REF}" --format '%(contents:body)' | grep . | tee release-body
-
       - name: Bootstrap automake
         run: ./autogen.sh
 
       - name: Build release
         run: make dist -j$(nproc)
 
       - name: Note source archive name
-        run: basename "$(make dump-dist)" | tee source-filename
+        id: source_filename
```

Comment:
I wonder if you'd like to drop this step and use something like `archive: cockpit-${{ github.ref }}.tar.xz` or so?

Reply:
`github.ref` is something like `refs/tag/123.4`. So this needs some extra string mangling using jinja2 API. I haven't done this before, let me check if there is something.

Reply:
I dug a bit, and I didn't find anything.

Reply:
Done -- it's using `github.ref_name` now.